### PR TITLE
Fix/edge weight

### PR DESCRIPTION
**Pull request by borismattijssen**
Fixes the scale of the edge weight after filtering.
Before the fix, it would still scale to the total amount of genomes in the data set instead of the total amount of genomes currently in the filtered data.

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java [hunk 1]
```diff
@@ -169,7 +169,8 @@ private void addNormalEdge(Graph graph, Wrapper from, Wrapper to, int i) throws
 		Edge edge = graph.addEdge(from.getId() + "-" + to.getId(),
 				Integer.toString(from.getId()), Integer.toString(to.getId()), true);
 		int weight = from.getOutgoingWeight().get(i);
-		float percent = ((float) weight) / ((float) zoomedGraphModel.getGenomes().size());
+		float percent = ((float) weight) / ((float) zoomedGraphModel.getGenomesCount());
+//		System.out.println(zoomedGraphModel.getGenomesCount());
```
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: Commented code

**Comment by Delfil** [comment]
All ok, can be merged

**Comment by kwendel** [comment]
Good for merging after build error fix!

**Comment by Delfil** [comment]
Buildfailure was not caused by the changed made.

**Comment by Delfil** [comment]
Kwender: "Build failure was caused by old version of travis"

**Pull request merged**